### DISCUSSION TITLE
Does LoadImage returns arrays in HU units?

### DISCUSSION THREAD

**Discussion by achoruzy** [Project-MONAI/MONAI#3564] [General]
Hi,

Couldn't find any information about that in the monai docs or here as well as in ITK docs. Maybe someone may clarify that one.

Does monai's LoadImage transform returns Dicoms and Nifti as already converted to HU units arrays?

From what I see, assume yes, but being sure about that would be nice and beneficial.

**Comment by Nic-Ma** [comment]
Hi @industArk ,

Thanks for your interest here.
I think we don't convert the data values in the DICOM or NIfTI files, just load the info of header into meta data and load the raw data into numpy array.
@ericspod Please feel free to correct me if I misunderstood it.

Thanks.

**Comment by wyli** [comment]
If the question is about applying the intensity rescaling with slope and intercept, it is done for nifti and dicom automatically.

**Comment by achoruzy** [comment]
Thank you, gents.
@wyli yes, that's what I asked about. Thank you for your clarification.
